Android/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required (VERSION 2.8.7)
+project(MCServer)
+include(ExternalProject)
+
+externalproject_add(lua_native
+	SOURCE_DIR ../lib/lua
+	CMAKE_GENERATOR "Visual Studio 14 2015"
```

Comment:
> “is there any reason not to use...”
Did you mean to use? If you did, I think it didn’t work if I didn’t specify that variable, but I’ll try again soon.